crates/bevy_ecs/src/system/commands/command_queue.rs

```diff
@@ -69,6 +71,36 @@ impl CommandQueue {
         std::mem::forget(command);
     }
 
+    /// Push a [`BoxedCommand`] onto the queue.
+    #[inline]
+    pub fn push_boxed(&mut self, boxed_command: BoxedCommand) {
+        let size = std::mem::size_of_val(boxed_command.command.deref());
+        let old_len = self.bytes.len();
```

Comment:
nit. Everything below is duplicated from push function above, this should be refactored in a utility function, esp as it duplicates unsafe code

Reply:
I think it's different enough now, and will continue to be different enough, to where having a common utility fn could be more of a pain.